.gitattributes

```diff
@@ -1,2 +1,2 @@
-* text eol=lf
+* text=auto
```

Comment:
for some reason `eol=lf` was doing something weird to `gradle/wrapper/gradle-wrapper.jar` and was causing it not to be usable. this caused the previous CI failure. changing this to `text=auto` fixed it.

Reply:
This change will break the tests when run on a windows machine.  Maybe instead add a line marking gradle-wrapper.jar as binary?

Reply:
From the documentation it seems that auto should automatically set the line endings to LF when git detects that it is a text file. However, I don't have a windows machine to test this one. Do you?
> Set to string value "auto"
>    When text is set to "auto", the path is marked for automatic end-of-line conversion. If Git decides that the content is text, its line endings are converted to LF on checkin. When the file has been committed with CRLF, no conversion is done.

From: https://www.kernel.org/pub/software/scm/git/docs/gitattributes.html

Reply:
Looks like you're right, but I'd still rather not depend on Git's auto-detection.

Right now it's this, which has worked for a long time:

```
* text eol=lf
*.png binary
```

This seems like the safest bet:

```
* text eol=lf
*.png binary
*.jar binary
```

Reply:
Ok, i'll change it then.